WcaOnRails/app/views/server_status/index.html.erb

```diff
@@ -2,31 +2,131 @@
 <div class="container">
   <h1><%= yield(:title) %></h1>
 
-  <% if @everything_good %>
+  <% if @everything_good && @total_missing_outdated == 0 %>
     <%= alert :success, "We're fine. We're all fine here, now, thank you. How are you?" %>
+  <% elsif @everything_good %>
+    <%= alert :warning, "The server is alright, but some translations would enjoy an update." %>
   <% else %>
     <%= alert :danger, "Check below, something may be wrong." %>
   <% end %>
+<%
+    kind_class = if @oldest_job_that_should_have_run_by_now
+                   "danger"
+                 else
+                   "success"
+                 end
+%>
```

Comment:
This is kinda long, I'd suggest either doing it with ternary or moving this to a view helper.